lib/menu.rb

```diff
@@ -0,0 +1,13 @@
+class Menu
+  def initialize(*args)
+    @dishes = args
+  end
+
+  def view_menu()
+    output_str = ""
+    @dishes.each{ |dish|
+      output_str << "#{dish.name}: £#{sprintf('%.2f', dish.price)}\n"
+    }
+    return output_str
+  end
+end
```

Comment:
could put output_str in initialize method to make view_menu shorter but reads well anyway